recipe/build-grpcio.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+export GRPC_BUILD_WITH_BORING_SSL_ASM=""
+export GRPC_PYTHON_BUILD_SYSTEM_CARES="True"
+export GRPC_PYTHON_BUILD_SYSTEM_GRPC_CORE="True"
+export GRPC_PYTHON_BUILD_SYSTEM_OPENSSL="True"
+export GRPC_PYTHON_BUILD_SYSTEM_RE2="True"
+export GRPC_PYTHON_BUILD_SYSTEM_ZLIB="True"
+export GRPC_PYTHON_BUILD_WITH_CYTHON="True"
+export GRPC_PYTHON_USE_PREBUILT_GRPC_CORE=""
+
+if [[ "${target_platform}" == linux-* ]]; then
+    # set these so the default in setup.py are not used
+    # it seems that we need to link to pthrad on linux or osx.
```

Comment:
```suggestion
    # it seems that we need to link to pthread on linux or osx.
```

Does the "or osx" part really apply here?